core/src/test/scala/com/banno/kafka/ConsumerAndProducerApiSpec.scala

```diff
@@ -1,6 +1,7 @@
 package com.banno.kafka
 
-import cats.syntax.all._
+import org.scalacheck._
+import cats.implicits._
```

Comment:
FYI we should be using `cats.syntax.all._` where we can to ease the transition to Scala 3, where different rules for given imports will result in the deprecation of this namespace.